### ISSUE TITLE
Youtube Stream

### ISSUE THREAD

**Issue by robotrono** [Splamy/TS3AudioBot#259]
Hmm Im trying to play a YT Stream Music and works fine but after 4 hrs the bot is stoped
Log:

```
18:44:43.8232|DEBUG Got message from Akira: !play [URL]https://www.youtube.com/watch?v=29Ai54uxB48[/URL]
18:44:43.8322|DEBUG User Akira created session with the bot
18:44:44.1014|DEBUG Falling back to youtube-dl!
18:44:46.7651|DEBUG youtube-dl succeeded!
18:44:46.7778|DEBUG AudioResource start: youtube ID:29Ai54uxB48
00:02:46.0182|DEBUG Got message from McCloud: !su
00:02:46.0196|DEBUG User McCloud created session with the bot
00:34:47.3746|DEBUG Got message from McCloud: !un
00:44:52.2616| INFO Song queue ended: No next song could be played.
```

**Comment by Splamy** [comment]
Lol, I'm surprised that streams even work at all. Last time I checked it didn't work and I opened a issue for it #58 
It probably has the same problem as all live stream soures that even a shot drop will cause the song to end.
I'll try to get a dedicated stream mode which will handle those things more sophisticated.
For now as a workaround I can suggest you to enable playlist looping with `!loop on` so the bot will (hopefully) restart when the connection drops.
